editor/editor_file_system.h

```diff
@@ -37,6 +37,9 @@
 #include "core/templates/safe_refcount.h"
 #include "core/templates/set.h"
 #include "scene/main/node.h"
+
+#include "core/templates/thread_work_pool.h"
```

Comment:
There's no reason to separate this from the rest, can be with other core includes.